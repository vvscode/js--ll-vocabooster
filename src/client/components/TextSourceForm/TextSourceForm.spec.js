import React from 'react';
import { shallow } from 'enzyme';
import Component from './TextSourceForm';

describe('TextSourceForm', () => {
  let props;
  let getShallowComponent;

  beforeEach(() => {
    props = {};

    getShallowComponent = () => shallow(<Component {...props} />);
  });

  describe('render', () => {
    it('should render with all props', () => {
      expect(getShallowComponent()).toMatchSnapshot();
    });
  });
});
