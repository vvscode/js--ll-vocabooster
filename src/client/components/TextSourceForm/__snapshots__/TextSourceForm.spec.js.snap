// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextSourceForm render should render with all props 1`] = `
<Form
  as="form"
  onSubmit={[Function]}
>
  <FormField
    control={[Function]}
    label="Source text"
    name="sourceText"
    onChange={[Function]}
    placeholder="Source of your new words ..."
  />
  <FormField
    control={[Function]}
  >
    Submit
  </FormField>
</Form>
`;
